osh.spec

```diff
@@ -234,15 +234,19 @@ fi
 %{_sbindir}/osh-retention
 %{_sbindir}/osh-stats
 %{_sysconfdir}/osh/hub
+%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
+%exclude %{python3_sitelib}/osh/hub/scripts/umb-emit.py
 %{python3_sitelib}/osh/hub
 %{_unitdir}/osh-retention.*
 %{_unitdir}/osh-stats.*
 %exclude %{python3_sitelib}/osh/hub/settings_local.py*
 %exclude %{python3_sitelib}/osh/hub/__pycache__/settings_local.*
+# These file should be readable and writable by respective groups.
 %dir %attr(775,root,root) %{_localstatedir}/log/osh
 %dir %attr(775,root,apache) %{_localstatedir}/log/osh/hub
-%ghost %attr(640,apache,apache) %{_localstatedir}/log/osh/hub/hub.log
 %attr(775,root,apache) %{_sharedstatedir}/osh/hub
```

Comment:
Here you can utilize defattr too:
```
%defattr(644, root, apache)
```
And everything below this line will have this owner and attributes.